core.lua

```diff
@@ -1,27 +1,37 @@
 BuffDuty = LibStub("AceAddon-3.0"):NewAddon("BuffDuty", "AceConsole-3.0")
 local CHAT_COMMAND = "buffduty"
+local MESSAGE_COMMAND = "buffduty-msg"
 local defaults = {
     factionrealm = {
-        duties_cache = {}
+        duties_cache = {},
+        custom_messages = {}
     }
 }
 
 function BuffDuty:OnInitialize()
     self:init()
+
+    BuffDuty.Messages:Initialise()
 end
 
 function BuffDuty:init()
     self:RegisterChatCommand(CHAT_COMMAND, "Command")
+    self:RegisterChatCommand(MESSAGE_COMMAND, "CommandMessage")
 
     --Init DataBase
     self.db = LibStub("AceDB-3.0"):New("BuffDutyDB", defaults)
     BuffDuty.Cache.duties_cache = self.db.factionrealm.duties_cache
+    BuffDuty.Messages.custom_messages = self.db.factionrealm.custom_messages
+end
+
+function BuffDuty:OnEnable()
+    BuffDuty.Messages:Load()
 end
 
 local function executeLogic(input)
     -- Checks whether makes sense to assign people
     if (GetNumGroupMembers() < 10) then
-        BuffDuty:printInfoMessage("Current Group/Raid is too small. No sense in assigning buffs.")
+        BuffDuty.printInfoMessage("Current Group/Raid is too small. No sense in assigning buffs.")
```

Comment:
I remember that i put " : " for a specific reason. Probably arguments were not parsing as I expected. Take a look at [this ](https://stackoverflow.com/questions/4911186/difference-between-and-in-lua)to make sure you are using . correctly.

Reply:
That's why I changed it... printInfoMessage has no need for `self` so there is no need to confuse things by calling it with a `:`